ush/wave_grib2_sbs.sh

```diff
@@ -35,6 +35,8 @@
   export LOUD=${LOUD:-YES}; [[ $LOUD = yes ]] && export LOUD=YES
   [[ "$LOUD" != YES ]] && set +x
 
+  WGRIB2=${WGRIB2:-${WGRIB2_ROOT:-${wgrib2_ROOT:?}}/bin/wgrib2}
```

Comment:
I'm assuming this. was needed because hpc-stack no longer defines this? Should this be added in a job or higher level script instead?

Reply:
Yes, although I should just add it to the modulefile instead (I did this for a different reason on WCOSS). I'll make an update.

Reply:
FWIW, I think hpc-stack is supposed to be adding the WGRIB2 variable at some point.